src/control/cmd/dmg/README.md

```diff
@@ -1,286 +1,66 @@
-<p>
+## Certificates
 
-```bash
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/dmg --help
-Usage:
-  dmg [OPTIONS] [command]
+Local certificate locations can be specified in the
+[client config file](/utils/config/daos.yml).
 
-Application Options:
-  -l, --hostlist=    comma separated list of addresses <ipv4addr/hostname:port> (default: localhost:10001)
-  -f, --hostfile=    path of hostfile specifying list of addresses <ipv4addr/hostname:port>, if specified takes preference over HostList
-  -o, --config-path= Client config file path
+For more details on how certificates are used within DAOS, see the
+[Security documentation](/src/control/security/README.md#certificate-usage-in-daos).
 
-Help Options:
-  -h, --help         Show this help message
+## Communications
 
-Available commands:
-  network  Perform tasks related to locally-attached network devices (aliases: n)
-  pool     Perform tasks related to DAOS pools (aliases: p)
-  service  Perform distributed tasks related to DAOS system (aliases: sv)
-  storage  Perform tasks related to locally-attached storage (aliases: st)
+The management tool is a gRPC client and uses the
+[management API](/src/control/client) to interact concurrently with
+[`daos_server`](/src/control/cmd/daos_server/README.md) processes
+running listening gRPC servers that host the control service.
 
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/dmg storage --help
-Usage:
-  dmg [OPTIONS] storage <list>
+Communications take place over the management network and addresses of
+remote storage servers to connect to can be specified on the commandline
+or in the [client config file](/utils/config/daos.yml).
 
-Application Options:
-  -l, --hostlist=    comma separated list of addresses <ipv4addr/hostname:port> (default: localhost:10001)
-  -f, --hostfile=    path of hostfile specifying list of addresses <ipv4addr/hostname:port>, if specified takes preference over HostList
-  -o, --config-path= Client config file path
+For details on how gRPC communications are secured and authenticated, see the
+[Security documentation](/src/control/security/README.md#host-authentication-with-certificates).
 
-Help Options:
-  -h, --help         Show this help message
+## Functionality
 
-Available commands:
-  list  List locally-attached SCM and NVMe storage (aliases: l)
+The functionality provided by the management tool is split into
+domains which map to individual subcommands.
 
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/dmg service --help
-Usage:
-  dmg [OPTIONS] service <kill-rank>
+For exhaustive subcommand listings please see output of
+`dmg [<subcommand>] --help`.
 
-Application Options:
-  -l, --hostlist=    comma separated list of addresses <ipv4addr/hostname:port> (default: localhost:10001)
-  -f, --hostfile=    path of hostfile specifying list of addresses <ipv4addr/hostname:port>, if specified takes preference over HostList
-  -o, --config-path= Client config file path
+### Network
```

Comment:
I'm wondering if it might be better to offer a basic description of the functionality offered by the subcommand, rather than listing a subset of the actual command. Especially since we're already pointing at the help above. @dpquigl is also working on adding a manpage for dmg, which we could point at from here for more exhaustive help, when it's available.